Dockerfile

```diff
@@ -1,26 +1,36 @@
 FROM heroku/cedar:14
 MAINTAINER Terence Lee <terence@heroku.com>
 
+ENV RUBY_MAJOR 2
+ENV RUBY_MINOR 3
+ENV RUBY_PATCH 0
```

Comment:
Fair enough. RE `RUBY_BASE`, when submitting this PR I hadn't seen https://github.com/heroku/docker-ruby/pull/8. Is that going to be accepted?